projects/core/src/features-config/feature-toggles/feature-toggles-tokens.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, InjectionToken, inject } from '@angular/core';
+
+/**
+ * Properties for gradually enabling/disabling breaking changes introduced in Spartacus.
+ *
+ * Feature toggles facilitate the incremental evolution of the Spartacus product,
+ * allowing Spartacus users to adjust to breaking changes over time.
+ *
+ * Each feature toggle corresponds to a specific breaking change introduced
```

Comment:
`query:` Wasn't the `FeatureToggles` mechanism supposed to be responsible for enabling/disabling features in general? I thought the mechanism, which is already in the code, would be responsible for breaking changes. I would appreciate being straightened out if I am wrong :)

Reply:
Updated text. What do you think?
```
/**
 * Properties for gradually enabling/disabling breaking changes introduced in Spartacus.
 *
 * Feature toggles facilitate the incremental evolution of the Spartacus product,
 * allowing Spartacus users to adjust to breaking changes over time.
 *
 * Each feature toggle corresponds to a specific breaking change introduced
 * in a Spartacus release.
 *
 * Some feature toggles might be only temporary, with a defined duration
 * (specified in the official documentation), after which the related
 * breaking change will be permanently enabled.
 */
 ```

Reply:
Updated text again and shortened, to be more reader friendly
```
/**
 * Properties for gradual adaptation to breaking changes in Spartacus via feature toggles.
 * Each toggle has its own lifespan defined in the official docs.
 * It should be provided in the root injector (in an eagerly loaded module).
 */
```